packages/web-pkg/src/composables/piniaStores/webWorkers.ts

```diff
@@ -0,0 +1,52 @@
+import { defineStore } from 'pinia'
+import { Ref, ref, unref } from 'vue'
+import { v4 as uuidV4 } from 'uuid'
+import { UseWebWorkerReturn, useWebWorker } from '@vueuse/core'
+
+export interface WebWorker<T = any> extends UseWebWorkerReturn<T> {
+  id: string
+}
+
+export type WorkerTopics = 'startProcess' | 'tokenUpdate'
```

Comment:
```suggestion
export type WorkerTopic = 'startProcess' | 'tokenUpdate'
```

You're not defining an enum but a type, so the var name should be singular (I know, ultra nitpick, sorry).